.github/actions/tests/run-e2e-tests/action.yml

```diff
@@ -6,6 +6,11 @@ inputs:
     report-name:
         description: Name of Allure report to be generated.
         required: true
+    tests:
+        description: Specific tests to run, separated by single whitespace. See https://playwright.dev/docs/test-cli
+    playwright-config:
+        description: The Playwright configuration file to use.
+        default: playwright.config.js
```

Comment:
We got 2 playwright config files at the moment. This `playwright-config` input allows us to specify which of these config files would be used.

On a side note, perhaps there's a more graceful way of running the release tests without needing a second PW config 🤔